docs/pvacseq/errors.rst

```diff
@@ -0,0 +1,169 @@
+Common Errors
+-------------
+
+Input VCF Sample Information
+____________________________
+
+**VCF contains more than one sample but sample_name is not set.**
+
+pVACseq supports running with a multi-sample VCF as input. However, it still
+requires the user to pick the sample to analyze, as only variants that are
+called in a sample will be processed.
```

Comment:
If we're saying it's limited to the chosen sample, I'd make that more clear.  Variants called only in other samples were called in `a` sample...just not the one we care about.
```suggestion
called in the specified sample will be processed.
```